outputs.tf

```diff
@@ -4,10 +4,10 @@ output "private_endpoints" {
 }
 
 output "resource" {
-  depends_on = [
-    azurerm_key_vault_key.this,
-    azurerm_key_vault_secret.this
-  ]
+  # depends_on = [
```

Comment:
Minor one, but best to remove commented code if we can.

Reply:
Good shout